elyra/pipeline/pipeline.py

```diff
@@ -102,6 +103,22 @@ def include_subdirectories(self):
     def env_vars(self):
         return self._env_vars
 
+    @property
+    def env_vars_as_dict(self) -> Dict:
```

Comment:
Please change the name to not include the resulting datatype information.  It can be inferred from the typing hint and docstrings.
```suggestion
    def env_vars(self) -> Dict:
```

Reply:
env_vars is already defined and is not returning a dict.

Reply:
OK - I would say that the constructor should convert the `list` to a `dict `and never expose `env_vars` as a list - it's really not useful in that form.  In fact, it would reduce [this block of code](https://github.com/elyra-ai/elyra/blob/master/elyra/pipeline/processor_kfp.py#L223-L230) to
```python
    if operation.env_vars:
        pipeline_envs.update(operation.env_vars)
```
We ONLY use the envs_vars as a dictionary - practically speaking.

Reply:
this has implications all way to bootstrap script,  so I would prefer to do this in a separate PR if you are ok with it

Reply:
Ok - that's fine.